### PR TITLE
More copy-edits to readme.

### PR DESCRIPTION
**Pull request by salimfadhley**
Fix the FLW tags as per steve's suggestion

### CODE REVIEW

**Comment by stestagg** [comment]
Candidatw

**Comment by salimfadhley** [comment]
will fix :-(

**Comment by stestagg** [comment]
:)

File: README.md [hunk 1]
```diff
@@ -82,14 +86,9 @@ It's a static web-site. Any time a GitHub pull request is merged it will automat
...
-### I'm an (agent|headhunter|recruitment consultant), can I use this site?
-
-Yes, but provided you are clear about the following details:
-
-* You need to be clear about the company you are recruiting for. So for example you cannot give "Top Tier Investment Bank" or "Awesome Media Agency" as the name of the company.
-* You need to include the name of your agency
+### I'm an (agent|headhunter|recruitment consultant|IT Consultancy), can I still use this site?
 
-... And you need to learn how to do a GitHub pull request. 
+Yes, but you must include the name of your client - where the candidatw will ultimately be working. If you don't include this information your job advert may be rejected or deleted.
```
Comment (lordmauve) [inline README.md]: Typo.

**Pull request merged**